shared/common-adapters/input.desktop.js

```diff
@@ -114,7 +114,6 @@ export default class Input extends Component {
           rows={this.props.rows}
           rowsMax={this.props.rowsMax}
           style={{...textStyle, ...globalStyles.flexBoxColumn, ...this.props.textStyle}}
-          textAlign='center'
```

Comment:
does this affect the alignment? that error is inside material ui and not our code so its possible the bug needs to be fixed there.